doc/userguide/rules/header-keywords.rst

```diff
@@ -439,7 +439,7 @@ Example of the itype keyword in a signature:
 The following lists all ICMP types known at the time of writing. A recent table can be found `at the website of IANA <https://www.iana.org/assignments/icmp-parameters/icmp-parameters.xhtml>`_
 
 ==========  ==========================================================
-ICMP Type   Name
+ICMP Type  Name
 ==========  ==========================================================
```

Comment:
Make sure the `=` before and after are aligned properly.

Reply:
I've fixed this up.